src/common/Widget.js

```diff
@@ -578,6 +579,29 @@
     };
 
     //  DOM/SVG Node Helpers  ---
+    
+    Widget.prototype.showBusy = function (show) {
+        if (show) {
+            //show the indicator
+        } else {
+            //hide the indicator
```

Comment:
@mlzummo @jbrundage Here is my .02¢ on the subject:

We could make a Loader widget that utilizes the built in FontAwesome spinner icons https://fortawesome.github.io/Font-Awesome/examples/ (scroll down to "Animated Icons").  With this approach we simply utlilize the FaCar widget, its all code, there are no extra files to worry about and track.  There are several different options and speed/size etc. can all be configured.  We can allow user to over ride with their own image if they choose to.